cfunits/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: cfunits
+  version: {{environ['CBUILD_CFUNITS_VERSION']}}
```

Comment:
Our style is to avoid environment variable to compose the `yaml` file to increase the readability.  We want our users to browse the recipes and know what gets build.

How hard would it be to change these?